addons/mail/static/src/core/common/composer.js

```diff
@@ -87,8 +87,10 @@ export class Composer extends Component {
     setup() {
         super.setup();
         this.isMobileOS = isMobileOS();
-        this.SEND_KEYBIND_TO_SEND = markup(
-            _t("<samp>%(send_keybind)s</samp><i> to send</i>", { send_keybind: this.sendKeybind })
+        this.OR_PRESS_SEND_KEYBIND = markup(
+            _t("or press %(send_keybind)s", {
+                send_keybind: this.sendKeybinds.map((key) => `<samp>${key}</samp>`).join(" + "),
+            })
```

Comment:
This looks over-complicated to me, why do we need an array and a join when the string can be crafted easily inside the (useless?) getter?

Reply:
or press send_keybind is a locale, so this needs `_t()`.
The string has few custom styling with `<samp>` around keybinds for improved readability.

The array join is so that there's a single point to define keybinds, each configuration is an array of keybinds: `[ENTER]` & [`CTRL, ENTER]`.

Reply:
Yes, this explains the code but it does not explain why the following code could not work. It's simpler since you avoid the `.map` and the `.join`. 

```javascript
   get sendKeybinds() {
        return this.props.mode === "extended" ? _t("CTRL") + " + " + _t("Enter") : _t("Enter");
    }
```

Reply:
It felt best to put most string content in `OR_PRESS_SEND_KEYBIND`, including the `" + "`.
`sendKeybinds` concern is not formatting but telling which keybinds are active for the composer at hand.
Doing so let each LOCs manage their own business.

Reply:
For potential users of `sendKeybinds`, this gives the shortcut to post the message.
If we put the whole string, they'd now have to dissect a string that becomes translated.
I hope you agree this is a much worse change for code that could depend on this getter, hence why I wanted to keep the semantics of the getter as the safest change in this PR.